app/helpers/effective_datatables_private_helper.rb

```diff
@@ -90,12 +90,13 @@ def datatable_search_tag(datatable, name, opts)
 
     collection = opts[:search].delete(:collection)
     value = datatable.state[:search][name]
-
     input_html = {
       name: nil,
       value: value,
       title: title,
       pattern: pattern,
+      autocomplete: opts[:search][:autocomplete] || SecureRandom.hex(32),
```

Comment:
I don't understand this autocomplete default value.

What does the `SecureRandom.hex(32)` do here?

Imho this is an invalid value, as per

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Is there something I'm missing? :)